docker/ceph/ci/pre-commit.sh

```diff
@@ -1,13 +1,13 @@
 #!/bin/bash
 
-set -e
+set -ex
 
 echo 'Running pre-commit hook...'
 
 # shellcheck disable=SC1091
 source /docker/ci/sanity-checks.sh
 
-NPM_PACKAGE_FILES=$(git diff --cached --name-only --diff-filter=ACMRTUXB | grep -E "package(-lock){0,1}.json" -c)
+NPM_PACKAGE_FILES=$(git diff --cached --name-only --diff-filter=ACMRTUXB | grep -E "package(-lock){0,1}.json -c" || true)
```

Comment:
the `-c` should be outside double-quotes right?
```suggestion
NPM_PACKAGE_FILES=$(git diff --cached --name-only --diff-filter=ACMRTUXB | grep -E "package(-lock){0,1}.json" -c || true)
```

Reply:
Yup, it's an option of grep
```suggestion
NPM_PACKAGE_FILES=$(git diff --cached --name-only --diff-filter=ACMRTUXB | grep -Ec "package(-lock)?.json" || true)
```